docs/TheThingsNetwork.md

```diff
@@ -64,7 +64,7 @@ RX Delay 1: 1000
 RX Delay 2: 2000
 ```
 
-See the [DeviceInfo](https://github.com/TheThingsNetwork/arduino-device-lib/blob/master/examples/DeviceInfo/DeviceInfo.ino) example.
+See the [DeviceInfo](../examples/DeviceInfo/DeviceInfo.ino) example.
```

Comment:
Please leave the absolute paths; we're also importing this Markdown in external documentation that are unable to handle the relative path.

Reply:
Thanks for the feedback. I reverted the relative URLs to the absolute Github ones.